client/my-sites/comments/comment-tree/index.jsx

```diff
@@ -81,7 +81,7 @@ export class CommentTree extends Component {
 
 	getCommentsPage = ( comments, page ) => {
 		const startingIndex = ( page - 1 ) * COMMENTS_PER_PAGE;
-		return slice( comments, startingIndex, startingIndex + COMMENTS_PER_PAGE );
```

Comment:
While `comments` is not required it is also treated as always present by the rest of the component (see the line where it calls `comments.length` without a null check) so I believe this safe-ish.

Reply:
Indeed, thanks for verifying though 👍